shared/settings/feedback/index.tsx

```diff
@@ -137,9 +137,20 @@ class Feedback extends React.Component<Props, State> {
                 />
               </Kb.Box2>
             )}
-            <Kb.ButtonBar>
-              <Kb.Button fullWidth={true} label="Send" onClick={this._onSendFeedback} waiting={sending} />
-            </Kb.ButtonBar>
+            {!this.props.loggedOut && (
+            <Kb.Box2 style={styles.btnContainer} direction="horizontal" gap="tiny">
+              <Kb.ButtonBar >
+                <Kb.Button label="Send" onClick={this._onSendFeedback} waiting={sending} fullWidth={true}/>
+              </Kb.ButtonBar>
+            </Kb.Box2>
+            )}
+            {this.props.loggedOut && (
+            <Kb.Box2 direction="horizontal" gap="tiny">
```

Comment:
instead of repeating this code
```suggestion
            <Kb.Box2 style={this.props.loggedOut ? undefined : styles.btnContainer} direction="horizontal" gap="tiny">
```
and then delete the extra copy and take it out of the conditional. Might also rename that style to make it clear what it's doing and/or put your GH comment as a comment on that style.

Reply:
Though, actually, I would do `alignSelf={this.props.loggedOut ? 'center' : 'flex-start'}` because we already have a prop for alignSelf on box2 and that removes a layer of abstraction of doing it through the styles.